esti/rollback_test.go

```diff
@@ -244,3 +245,49 @@ func TestRevert(t *testing.T) {
 	body := string(getObjResp.Body)
 	require.Equal(t, objContent2, body, fmt.Sprintf("path: %s, expected: %s, actual:%s", objPath2, objContent2, body))
 }
+
+func TestRevertConflict(t *testing.T) {
```

Comment:
minor: Could be a controller level test, it would reduce the runtime.

Reply:
I'm all in favour of **adding** a controller test, but do you think the `Esti` test should be removed?
As I see it, these E2E tests are the important ones, in terms of CI/CD, as they really cover a true use-case.
WDYT?

Reply:
Added `controller` UT and remove `Esti` test